scipy/stats/_continuous_distns.py

```diff
@@ -957,6 +958,8 @@ def _stats(self, c, d):
             lambda c, e1, e2, e3, e4, mu2_if_c: (
                 ((e4 - 4*e3*e1 + 6*e2*e1**2 - 3*e1**4) / mu2_if_c**2) - 3),
             fillvalue=np.nan)
+        if c.ndim == 0:
```

Comment:
If we remove the call of `broadcast_arrays` above, `c` might be a scalar without the `ndim` attribute.  We can get the same effect by using `np.ndim(c)`:
```suggestion
        if np.ndim(c) == 0:
```